fileio/ft_read_headshape.m

```diff
@@ -1346,6 +1346,36 @@
     shape.hex = shape.hex(:,1:8);
     shape.hex = shape.hex + 1; % this should be one-offset
 
+  case 'gmsh_binary'
+    [nodes, elements] = read_gmsh_binary(filename);
+    shape.pos = nodes.nodes(nodes.indx, :);
+
+    % this file format may contain a mixture of differently shaped elements
+    fnames = fieldnames(elements);
+    for k=1:numel(fnames)
+      el = elements.(fnames{k});
+      switch fnames{k}
+        case 'lines'
+          shape.line = el;
+        case 'triangles'
+          shape.tri = el;
+        case 'tetrahedra'
+          shape.tet = el;
+        case 'hexahedra'
+          shape.hex = el;
+        case 'lines_tag'
+          shape.tag_line = el;
```

Comment:
those are the scalar numeric fields (actually Nelementsx2 in the simnibs files that contain the tissue indices

Reply:
I would have a preference for something that is more generic and simpler. Is it possible (or likely) to have multiple tags in one structure, i.e. both `tag_line` and  `tag_hex`?

For neurojson_jmesh I decided that it is unlikely to have multiple tags (or multiple geometries that are based on the same vertices) and hence I named the output field simply `mesh.tag` (or color, normal, size, texture, value), where `size(mesh.tag,1)` can be used to determine whether it belongs to the vertices or to the elements (line/tri/tet/hex).   

See https://github.com/fieldtrip/fieldtrip/blob/de09d349ce524e3ebd726da62196107e6c24e34e/fileio/ft_read_headshape.m#L1382

Would that also work here?

Reply:
no, it would only work if this function 'decides' either to keep the volume elements, or the triangle elements. By construction the file contains both, and the tags refer to the elements, not the vertices. Right now, ft_read_headmodel makes this choice, based on the user specified 'meshtype', option. We could move this functionality from there into ft_read_headshape.